fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -193,6 +205,11 @@ public void writeFDBLuceneFileReference(@Nonnull final String name, @Nonnull fin
      * @param value the data to be stored
      */
     public void writeData(long id, int block, @Nonnull byte[] value) {
+        FDBStoreTimer timer = context.getTimer();
+        if (timer != null) {
+            //This may not be correct transactionally
+            timer.increment(FDBStoreTimer.Counts.LUCENE_DATA_SIZE, value.length);
```

Comment:
I still think this should be `LUCENE_WRITE_SIZE` - it is useful to know the type of operation for which the size is being reported.

Reply:
done.